src/dvr/dvr_config.c

```diff
@@ -818,14 +818,14 @@ const idclass_t dvr_config_class = {
     {
       .type     = PT_U32,
       .id       = "removal-days",
-      .name     = N_("DVR File Removal Time (days)"),
+      .name     = N_("DVR file removal period (days)"),
```

Comment:
I think I said this last time, so I won't beat it to death - why is it "log retention" but "file removal"? Aren't they the same, i.e. "DVR file retention period"?